jcl/source/windows/JclPeImage.pas

```diff
@@ -6935,9 +6935,19 @@ function PeBorUnmangleName(const Name: string): string;
     Result := '';
 end;
 
-function PeIsNameMangled(const Name: string): TJclPeUmResult;
+function PeIsNameMangled(const Name: string): TJclPeUmResult; {$IF CompilerVersion >= 23}inline;{$IFEND}
```

Comment:
Do not use $IF with compiler version, use the jedi.inc symbols: `$IFDEF SUPPORTS_INLINE`